src/main/resources/migrations/00006_create_table_mandates.sql

```diff
@@ -0,0 +1,14 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_table-mandates
+CREATE TABLE mandates (
+    id BIGSERIAL PRIMARY KEY,
+    payer_id BIGSERIAL NOT NULL,
+    external_id VARCHAR(30) NOT NULL,
```

Comment:
We might need an index on external_id, but not right now it seems, so 👍

Reply:
Absolutely! I'll add it now